src/restruct_poc/beams/calculate_jacobian.hpp

```diff
@@ -12,9 +12,10 @@ namespace openturbine {
 struct CalculateJacobian {
     Kokkos::View<Beams::ElemIndices*>::const_type elem_indices;  // Element indices
     Kokkos::View<double***>::const_type shape_derivative;        // Num Nodes x Num Quadrature points
-    View_Nx7::const_type node_position_rotation;        // Node global position/rotation vector
+    Kokkos::View<double** [7]>::const_type
+        node_position_rotation;                         // Node global position/rotation vector
     Kokkos::View<double** [3]> qp_position_derivative;  // quadrature point position derivative
-    View_NxN qp_jacobian;                               // Jacobians
+    Kokkos::View<double**> qp_jacobian;                 // Jacobians
```

Comment:
How is this change doing anything different?

Reply:
This is a consistency change.  Everything else in the kernel uses `Kokkos::View<...>`, so it makes it easier to read and compare when going down the list of variables.